app/src/main/res/values/strings.xml

```diff
@@ -312,7 +312,7 @@
   <string name="profile_edit_allow_download_heading">Allow Download Access</string>
   <string name="profile_edit_allow_download_sub">User is able to download and delete content without Administrator password</string>
   <!-- ProfilePictureActivity -->
-  <string name="profile_picture_activity_title">Profile Picture</string>
+  <string name="profile_picture_activity_title" translatable="false">Profile Picture</string>
```

Comment:
Again, not a part of this PR. Ditto at other places.

Reply:
okk